resources/views/admin/pages/add_category.blade.php

```diff
@@ -0,0 +1,66 @@
+@extends('admin.master_layout')
+
+@section('pageTitle', 'Add Category')
+@section('content')
+<div class="container-fluid">
+
+    <!-- Page Heading -->
+    <div class="d-sm-flex align-items-center justify-content-between mb-4">
+        <h1 class="h3 mb-0 text-gray-800">  {{ __('messages.🔹 Add Category') }}  </h1>
```

Comment:
```suggestion
        <h1 class="h3 mb-0 text-gray-800">{{ __('messages.addCategory') }}</h1>
```
tránh dùng khoảng trắng trong key nha, fix tương tự chỗ khác.